data/json/effects_on_condition/dream_eocs.json

```
@@ -5,6 +5,108 @@
     "//": "used to prevent multiple dreams at the same sleep",
     "effect": [ { "math": [ "dream_counter", "=", "0" ] } ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_TROUBLED_SLEEP_FATIGUE_MESSAGES",
+    "//": "Prints a flavour text message to the log about being afraid of sleep if the character has entered the tired status, has the trubbled sleeper trait, and has not taken preventative medication. The EOC’s set to recur every 8 to 20 minutes.",
+    "global": true,
+    "recurrence": [ 480, 1200 ],
```

if you want the time to be readable, you can use plain text, like `8 m` or `8 minutes`
```suggestion
    "recurrence": [ "8 minutes", "20 minutes" ],
```